error: without commandline argument

```
$ vetgen
panic: runtime error: index out of range

goroutine 1 [running]:
main.(*VetGen).Run(0xc0000c4080, 0xc0000761b0, 0x0)
	/Users/vvakame/work/gopath/src/github.com/gostaticanalysis/vetgen/gen.go:31 +0x223
main.main()
	/Users/vvakame/work/gopath/src/github.com/gostaticanalysis/vetgen/main.go:16 +0x14b
```

--- a/gen.go
+++ b/gen.go
@@ -28,7 +28,7 @@ type VetGen struct {
 }
 
 func (g *VetGen) Run() error {
-	if len(g.Args[0]) == 0 {
+	if len(g.Args) == 0 {
 		return errors.New("subcommand must be specified: init, add")
 	}
 	cmd, args := g.Args[0], g.Args[1:]

